internal/cmd/access/access.go

```diff
@@ -0,0 +1,226 @@
+	secret, err := cfg.KubeClient.Static().CoreV1().Secrets(cfg.namespace).Get(cfg.Ctx, cfg.name, metav1.GetOptions{})
+	if err != nil {
+		fmt.Printf("Error getting secret: %v", err)
+		return nil, err
+	}
+
+	currentCtx := cfg.KubeClient.ApiConfig().CurrentContext
+	clusterName := cfg.KubeClient.ApiConfig().Contexts[currentCtx].Cluster
+
+	// Create a new kubeconfig
+	kubeconfig := &api.Config{
+		Kind:       "Config",
+		APIVersion: "v1",
+		Clusters: map[string]*api.Cluster{
+			clusterName: {
+				Server:                   cfg.KubeClient.ApiConfig().Clusters[clusterName].Server,
+				CertificateAuthorityData: secret.Data["ca.crt"],
+			},
+		},
+		AuthInfos: map[string]*api.AuthInfo{
+			cfg.name: {
+				Token: string(secret.Data["token"]),
+			},
+		},
+		Contexts: map[string]*api.Context{
+			currentCtx: {
+				Cluster:   clusterName,
+				Namespace: cfg.namespace,
+				AuthInfo:  cfg.name,
+			},
+		},
+		CurrentContext: currentCtx,
+		Extensions:     nil,
+	}
+	return kubeconfig, nil
+}
+
+func createObjects(cfg *accessConfig) error {
+	err := createServiceAccount(cfg)
+	if err != nil {
+		return err
+	}
+
+	err = createSecret(cfg)
+	if err != nil {
+		return err
+	}
+
+	err = createClusterRole(cfg)
+	if err != nil {
+		return err
+	}
+
+	err = createClusterRoleBinding(cfg)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func createServiceAccount(cfg *accessConfig) error {
+	sa := v1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cfg.name,
+			Namespace: cfg.namespace,
+		},
+	}
+	_, err := cfg.KubeClient.Static().CoreV1().ServiceAccounts(cfg.namespace).Create(cfg.Ctx, &sa, metav1.CreateOptions{})
+	if client.IgnoreNotFound(err) == nil {
+		return err
+	}
+	return nil
+}
+
+func createSecret(cfg *accessConfig) error {
+	secret := v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cfg.name,
+			Namespace: cfg.namespace,
+			Annotations: map[string]string{
+				"kubernetes.io/service-account.name": cfg.name,
+			},
+		},
+		Type: v1.SecretTypeServiceAccountToken,
+	}
+	_, err := cfg.KubeClient.Static().CoreV1().Secrets(cfg.namespace).Create(cfg.Ctx, &secret, metav1.CreateOptions{})
+	if client.IgnoreNotFound(err) == nil {
+		return err
+	}
+	return nil
+}
+
+func createClusterRole(cfg *accessConfig) error {
+	cRole := rbacv1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cfg.clusterrole,
+			Namespace: cfg.namespace,
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{"", "extensions", "batch", "apps", "gateway.kyma-project.io", "servicecatalog.k8s.io"},
+				Resources: []string{"deployments", "replicasets", "pods", "jobs", "configmaps", "apirules", "serviceinstances", "servicebindings", "services", "secrets"},
+				Verbs:     []string{"create", "update", "patch", "delete", "get", "list"},
+			},
+		},
+	}
+	_, err := cfg.KubeClient.Static().RbacV1().ClusterRoles().Create(cfg.Ctx, &cRole, metav1.CreateOptions{})
+	if client.IgnoreNotFound(err) == nil {
```

Comment:
I would suggest to replace `client.IngoreNotFound` with `errors.IsAlreadyExists` from the `k8s.io/apimachinery/pkg/api/errors`. I think the `Create` func will never return a `not found` error because in this case this func will just create it